react/components/EditorContainer/StoreEditor/Styles/StyleEditor/typography/TypeToken.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { Dispatch } from 'react'
+import { RouteComponentProps } from 'react-router'
+import { Dropdown, Spinner } from 'vtex.styleguide'
+
+import { FormattedMessage } from 'react-intl'
+import ListFontsQuery, {
+  FontFamily,
+  ListFontsQueryResult,
+} from '../queries/ListFontsQuery'
+import StyleEditorHeader from '../StyleEditorHeader'
+import { prettify, prettifyCamelCase } from '../utils/typography'
+
+interface WrapperProps extends RouteComponentProps<TypeTokenParams> {
+  config: TachyonsConfig
+  onSave: () => void
+  updateConfig: (config: DeepPartial<TachyonsConfig>) => void
+}
+
+interface Props extends WrapperProps {
+  result: ListFontsQueryResult
+}
+
+const TypeTokenWrapper: React.FunctionComponent<WrapperProps> = props => {
```

Comment:
I recently found out that there is a shorthand version of `FunctionComponent`: `React.FC`, you could use that if you want

Reply:
Nice!